### PR TITLE
Add Actions support.

### PR DESCRIPTION
**Pull request by davidlehn**
- Add test, lint, and coverage workflow.
- Update README build status badge.
- Move coverage support to test/package.json.
- Rename test dir package name so npm doesn't complain about
  bedrock-test name conflict.

### CODE REVIEW

File: test/package.json [hunk 1]
```diff
@@ -1,12 +1,28 @@
 {
-  "name": "bedrock-test",
+  "name": "bedrock--test",
```
Comment (dlongley) [inline test/package.json:2]: This is going to be mistaken for a typo by someone at some point. Can we have a better name?

Comment (mattcollier) [reply test/package.json:2]: +1 to fixing that somehow.

Comment (davidlehn) [reply test/package.json:2]: I understand concern.  Suggestions?  This is an edge case just for bedrock itself.  npm didn't like installing `bedrock-test` inside of `bedrock-test`.  `__bedrock_test__`, `bedrock-test_`, `bedrock-t-e-s-t`?  Such a silly problem with silly solutions.

Comment (davidlehn) [reply test/package.json]: Using `bedrock-test-test`.  Good enough for now.
